apps/web-giddh/src/app/new-vs-old-Invoices/sales-by-person/sales-by-person.component.scss

```
@@ -0,0 +1,260 @@
+// ── Shared grid layout (4 columns: period flex, 3 fixed num) ─
+%sbp-expand-grid {
+    display: grid;
+    grid-template-columns: 1fr 120px 160px 120px;
+    align-items: center;
+    padding: 0 16px;
+}
+
+// ── Column header row ─────────────────────────────────────
+.sbp-expand-col-header {
+    @extend %sbp-expand-grid;
+    height: 36px;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)